certifier_service/certlib/certlib_proofs.go

```diff
@@ -1153,6 +1153,54 @@ func VerifySevAttestation(serialized []byte, k *certprotos.KeyMessage) []byte {
 		  byte dev_public_key[PUBLIC_KEY_SIZE];
 		};
  */
+
+/*
+ * TODO: Notes taken during call with Ye. To be deleted ...
+ *
+ * I need to probably do e2e implementation like this for CCA:
+ *
+ * Option 1: (John prefers this option)
+ *
+ *  - Figure out how to write this fn. How to verify evidence provided by CCA-shim?
+ *  - CCA-shim provides an attest function which generates an evidence report
+ *  - Need to write something to generate an evidence package on top of that evidence.
```

Comment:
I think this information is helpful not only for demo but also after demo.

- CCA attestation format
  - spec
     - section: A7.2.3 Attestation token format
     - url: https://developer.arm.com/documentation/den0137/latest 
  - code
     - url: https://git.trustedfirmware.org/TF-RMM/tf-rmm.git/tree/lib/attestation/include/attestation_token.h
- Platform key for signing realm is hard-coded
   - [private_key](https://github.com/ARM-software/arm-trusted-firmware/blob/master/lib/psa/delegated_attestation.c#L81)
   - public_key is encoded in attestation report and you can parse it by `islet_parse` API like below

--- 
## Get public key like [this line](https://github.com/vmware-research/certifier-framework-for-confidential-computing/blob/main/src/cca/cca_shim.cc#LL50C20-L50C41)
```c
rv = islet_parse("Realm signing public key", claims, claims_len, pub_key_out, pub_key_out_size);
```

Reply:
And also you can parse other fields in attestation report with the same way.
The field are listed at [here](https://github.com/Samsung/islet/blob/main/sdk/src/config.rs#LL39C16-L39C16).

+ We developed a CLI to explore attestation APIs.
If you use it you can explore without writing code.

I will upload this to Certifier soon.
https://github.com/Samsung/islet/tree/main/cli

Reply:
Again, @bitboom @jinbpark - I read your comment. But quite don't understand how you'd like me to update this code-based change-set.

If you wish this information be captured somewhere, it's probably best done in a new file, `instructions.md`, like we've recorded some design-level information for other platform's sample-apps.

For instance, see [simple_app_under_sev/instructions.md](https://github.com/vmware-research/certifier-framework-for-confidential-computing/blob/main/sample_apps/simple_app_under_sev/instructions.md), in the section titled: "General notes on building and running an SEV-enabled VM under KVM".

If you think the above design / interface type info has to be retained, I suggest we take it outside this change-set. And we can add it to a new instructions.md file for this app.

That file does not exist, yet. If you wish to take the lead on archiving that info, please do so; otherwise, let me know. And I'll try to find some time to do it.

Reply:
> If you think the above design / interface type info has to be retained

I don't think that info has to be retained in a new file `instructions.md`.  Just ignore my comment on this.